tests/unit_tests/test_general.py

```diff
@@ -0,0 +1,52 @@
+from hdmf.common import DynamicTable
+
+from nwbinspector import InspectorMessage, Importance
+from nwbinspector.checks.general import check_name_slashes, check_description
+
+
+def test_check_name_slashes_pass():
+    table = DynamicTable(name="test_name", description="")
+    assert check_name_slashes(obj=table) is None
+
+
+def test_check_name_slashes_fail():
+    """HDMF/PyNWB forbid "/" in the object names. Might need an external file written in MATLAB to test that?"""
+    for x in ["\\"]:
```

Comment:
why not test forward slash?

Reply:
See the description for that test; I'm not sure how to go about it using HDMF/PyNWB since the standard objects (table/time series) threw errors upon attempting to include a forward slash.

Reply:
ah

Reply:
it's also not possible with h5py.